GUI/src/cz/fidentis/gui/comparison_batch/ViewerPanel_Batch.java

```diff
@@ -365,32 +391,62 @@ private void canvas2MouseReleased(MouseEvent evt) {
     }
 
     private void canvas1MousePressed(java.awt.event.MouseEvent evt) {//GEN-FIRST:event_canvas1MousePressed
+        canvasClicked(evt, listener, canvas1);
+
+    }//GEN-LAST:event_canvas1MousePressed
+
+    private void canvasClicked(MouseEvent evt, ComparisonGLEventListener listener, Canvas canvas) {
```

Comment:
👍 pekne rozdelenie do funkcie, toto je presne jeden zo spravnych krokov k tomu aby sa to vsetko dalo robit len jeden krat a znovupouzit pre vsetky typy projektov